modules/src/mock/context.rs

```diff
@@ -833,9 +849,10 @@ impl ChannelKeeper for MockContext {
         timeout_height: Height,
         data: Vec<u8>,
     ) -> Result<(), Ics04Error> {
-        let input = format!("{:?},{:?},{:?}", timeout_timestamp, timeout_height, data);
-        self.packet_commitment
-            .insert(key, ChannelReader::hash(self, input));
+        self.packet_commitment.insert(
```

Comment:
I wonder if it would be better to change this trait method (i.e. `store_packet_commitment()` and others like it) so that we give it the precalculated commitment because we already allow users to define how the commitment is calculated. (cc: @romac, @plafer)